mithril-common/src/entities/block_range.rs

```diff
@@ -11,8 +11,7 @@ use crate::{
     StdResult,
 };
 
-/// BlockNumber is the block number of a Cardano transaction.
-pub type BlockNumber = u64;
+use super::BlockNumber;
```

Comment:
We should avoid `use super::` in production code, (it's fine for tests). can you replace it with a `use crate::entities::` instead ?

Reply:
Done